src/logger/src/metrics.rs

```diff
@@ -774,16 +796,18 @@ mod tests {
         }
 
         assert_eq!(
-            m2.count(),
+            metric.count(),
             M2_INITIAL_COUNT + NUM_THREADS_TO_SPAWN * NUM_INCREMENTS_PER_THREAD
         );
+    }
 
-        // Trying to store multiple values in the metric will result in keeping only the last
-        // value there.
-        m2.store(1);
-        m2.store(2);
-        assert_eq!(m2.0.load(Ordering::Relaxed), 2);
```

Comment:
why did you remove this small test?

Reply:
given the new implementation, I now see that this test doesn't make sense any more